src/ChartCreator/Options/OrderBy/OrderBy.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+import React, { useState, useEffect } from "react";
+import Grid from "@material-ui/core/Grid";
+import MenuItem from "@material-ui/core/MenuItem";
+import Select from "@material-ui/core/Select";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import FormControl from "@material-ui/core/FormControl";
+import InputLabel from "@material-ui/core/InputLabel";
+import RadioGroup from "@material-ui/core/RadioGroup";
+import Radio from "@material-ui/core/Radio";
+import { Typography } from "@material-ui/core";
+
+const buildOperations = ({ orderByColumns }) =>
+  orderByColumns.map(({ value, label }) => (
+    <MenuItem key={value} value={value}>
+      {label}
+    </MenuItem>
+  ));
+
+const OrderBy = ({ orderByColumns, orderBy, setOrderBy, isTimeType }) => {
+  const [columnId, setColumnId] = useState("");
+  const [order, setOrder] = useState("asc");
+  const [disabled, setDisabled] = useState(false);
+
+  useEffect(() => {
+    setDisabled(isTimeType);
+  }, [isTimeType]);
+
```

Comment:
`let disabled = isTimeType`
or
`let disabled = isTimeType || false` if default value is needed but i don't think
 and delete useEffect and useState?

Reply:
Agree with first one